### PR TITLE
A Cauldron of Doc fixes

### PR DESCRIPTION
**Pull request by dstrain115**
Miscellaneous doc fixes around cirq
    - Fix indenting where it is inappropriately detected as blockquotes
    - Fix references and commas at a few places.
    - Move some docstrings from init to class so that they show up on
    the API reference properly.
    - Fix list formatting
    - Fix one-line summaries that are more than one line.

### CODE REVIEW

File: cirq-core/cirq/linalg/transformations.py [hunk 1]
```diff
@@ -187,10 +187,13 @@ def targeted_conjugate_about(
     is a contraction between the given indices (indices for first $\cdot$,
     conj_indices for second $\cdot$).
 
-    More specifically this computes
-        $\sum tensor_{i_0,...,i_{r-1},j_0,...,j_{r-1}} *
+    More specifically, this computes:
+
+    $$i
```
Comment (dabacon) [inline cirq-core/cirq/linalg/transformations.py:192]: is this an extra "i"?

File: cirq-google/cirq_google/engine/engine_client.py [hunk 2]
```diff
@@ -88,10 +88,20 @@ def submit(self, func: Callable[..., Awaitable[_R]], *args, **kw) -> duet.Awaita
 
 
 class EngineClient:
-    """Client for the Quantum Engine API that deals with the engine protos and
-    the gRPC client but not cirq protos or objects. All users are likely better
-    served by using the Engine, EngineProgram, EngineJob, EngineProcessor, and
-    Calibration objects instead of using this directly.
+    """Client for the Quantum Engine API handling protos and gRPC client.
+
+    Thie is the client for the Quantum Engine API that deals with the engine protos
```
Comment (dabacon) [inline cirq-google/cirq_google/engine/engine_client.py:93]: Thie -> This

File: cirq-google/cirq_google/engine/engine_client.py [hunk 3]
```diff
@@ -88,10 +88,20 @@ def submit(self, func: Callable[..., Awaitable[_R]], *args, **kw) -> duet.Awaita
...
+    and the gRPC client but not cirq protos or objects. All users are likely better
+    served by using the `Engine`, `EngineProgram`, `EngineJob`, `EngineProcessor`, and
+    `Calibration` objects instead of using this directly.
+
+    Args:
```
Comment (dabacon) [inline cirq-google/cirq_google/engine/engine_client.py:98]: does this render correctly on doc site?  I thought you should keep the args in the init but put a description on the class for the best redering?

**Review by dabacon — APPROVED** [review]
couple of comments but otherwise LGTM

File: cirq-core/cirq/linalg/transformations.py (hunk 1 above)
Comment (dstrain115) [reply cirq-core/cirq/linalg/transformations.py]: Done.

File: cirq-google/cirq_google/engine/engine_client.py (hunk 2 above)
Comment (dstrain115) [reply cirq-google/cirq_google/engine/engine_client.py]: Done.

File: cirq-google/cirq_google/engine/engine_client.py (hunk 3 above)
Comment (dstrain115) [reply cirq-google/cirq_google/engine/engine_client.py:106]: Changed to this style.

**Pull request merged**